lib/cinch/plugins/wordgames/game.rb

```diff
@@ -3,17 +3,20 @@
 require_relative 'dict_word.rb'
 require_relative 'response.rb'
 
-
 module Cinch::Plugins::WordGames
   class Game
```

Comment:
This class is designed to be a "base" that the other wordgames will extend from (eg. Hangman) ... Please place the WordGame specific stuff into an extension of this base class and not in the class itself. In fact, the entire "wordgames" folder was made for this "base" purpose.